tests/data/cifar_10/source/resnet_cifar_10.py

```diff
@@ -106,21 +108,19 @@ def model_fn(features, labels, mode, params):
 
 
 def serving_input_fn(hyperpameters):
-    feature_spec = {INPUT_TENSOR_NAME: tf.FixedLenFeature(dtype=tf.float32, shape=(32, 32, 3))}
-    return tf.estimator.export.build_parsing_serving_input_receiver_fn(feature_spec)()
+    inputs = {INPUT_TENSOR_NAME: tf.placeholder(tf.float32, [None, 32, 32, 3])}
+    return tf.estimator.export.ServingInputReceiver(inputs, inputs)
 
 
-def train_input_fn(training_dir, hyperpameters):
-    return input_fn(tf.estimator.ModeKeys.TRAIN,
-                    batch_size=BATCH_SIZE, data_dir=training_dir)
+def train_input_fn(training_dir, hyperparameters):
+    return _input_fn(tf.estimator.ModeKeys.TRAIN, batch_size=BATCH_SIZE)
 
 
-def eval_input_fn(training_dir, hyperpameters):
-    return input_fn(tf.estimator.ModeKeys.EVAL,
-                    batch_size=BATCH_SIZE, data_dir=training_dir)
+def eval_input_fn(training_dir, hyperparameters):
+    return _input_fn(tf.estimator.ModeKeys.EVAL, batch_size=BATCH_SIZE)
 
 
-def input_fn(mode, batch_size, data_dir):
+def _input_fn(mode, batch_size):
```

Comment:
I think it'd be better if this method's name were a little more distinct from `input_fn`

Reply:
good point.